tests/vibrato_tests.ts

```diff
@@ -11,6 +11,7 @@ const VibratoTests = {
   Start(): void {
     QUnit.module('Vibrato');
     const run = VexFlowTests.runTests;
+    run('Vibrato with Bend', withBend);
```

Comment:
This test seems to be called twice. Delete one instance.

Reply:
oops...ok